public/app/plugins/panel/canvas/editor/panZoomHelp.tsx

```diff
@@ -11,7 +11,7 @@ export const PanZoomHelp = ({}: StandardEditorProps<string, unknown, unknown, un
 
   return (
     <>
-      <HorizontalGroup className={styles.hGroup}>
+      <Stack direction="row">
```

Comment:
Pics of this change... currently no way to pass a custom class down to Stack 🤔 

Before
<img width="548" alt="before" src="https://github.com/grafana/grafana/assets/22381771/c2245f74-7e5f-4eb6-86e6-8a29080f4146">

After
<img width="558" alt="after" src="https://github.com/grafana/grafana/assets/22381771/bb57fa1b-5992-47aa-a5b4-ce2f9ab49f52">